lib/handle-pull-request-change.js

```diff
@@ -1,6 +1,6 @@
 module.exports = handlePullRequestChange
 
-async function handlePullRequestChange (robot, context) {
+async function handlePullRequestChange (context) {
```

Comment:
removing it here is not enough, the robot argument is still passed at 

https://github.com/gr2m/wip-bot/blob/6d67f35f4168338a82b91885e16459e6aaf3df8c/index.js#L10

We'll have to remove the pre-binding of the robot argument there (remove `.bind(null, robot)`)

Reply:
Ah, dang. How did I forget :sweat_smile:

Reply:
Done the required changes, squashed the commits. :+1: